File: Python/1672_RichestCustomerWealth.py
from typing import List
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        maxWealth = 0
        for i in range(len(accounts)):
            wealth = 0
            for k in range(len(accounts[i])):
                wealth += accounts[i][k]
            if wealth > maxWealth:
                maxWealth = wealth
        return maxWealth

File: C#/1672_RichestCustomerWealth.cs
public class Solution {
    public int MaximumWealth(int[][] accounts){
        int maxWealth = 0;
        for(int i = 0; i < accounts.Length; i++){
            int wealth = 0;
            for(int k = 0; k < accounts[i].Length; k++){
                wealth += accounts[i][k];
            }
            if(wealth > maxWealth){
                maxWealth = wealth;
            }
        }
        return maxWealth;
    }
}